lldb/bindings/interface/SBValueDocstrings.i

```diff
@@ -45,6 +45,36 @@ SBValue instance which interprets the value object as representing the head of a
 linked list."
 ) lldb::SBValue;
 
+%feature("docstring", "
+Returns true if the SBValue holds any useful state
+and false otherwise.
+IsValid is a very weak API, lldb will only return
+invalid SBValues if it has no useful information
+about the SBValue.
+The two main ways you will end up with an invalid
+SBValue are:
+1) default constructed SBValues are not valid.
+2) SBValues that have outlived their SBTarget are
+no longer valid since its not safe to ask them
+questions.
+"
+) lldb::SBValue::IsValid
+
+%feature("docstring", "
+SBValues use the lldb::SBError object returned by
```

Comment:
I'm not sure "use" is the right verb here. My read is that it implies the SBValue does something with the SBError, but this function is the thing that "reports" the error right?
Suggestion:
"Returns an SBError object that contains information about the most recent error related to this SBValue." (followed by some examples like you have below)

Reply:
I did something between what was there and your suggestion.  I want to emphasize the point that you need to check whether your Value was successfully made, but then you also have to check every time lldb has to fetch a new value.  Putting those in sequence I thought made that point nicely.  See if this version is clearer.

Reply:
Looks good to me :)